lib/ReactViews/Drag/use-draggable.ts

```diff
@@ -0,0 +1,210 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+
+export const useDraggable = (options?: { handleSelector?: string }) => {
```

Comment:
nitpick: Should we convert `options` and `handleSelector` to required? AFAICT  `isValidDragHandle` test will fail unless this is set?

Reply:
if we don't specify handleSelector, it will allow the dragging of an entire element. In this case entire feature info panel would cause dragging effect